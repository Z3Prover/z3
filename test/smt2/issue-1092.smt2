; Copyright Microsoft 2017
; repro for GitHub Issue 1092

	(declare-datatypes () ((Destination (d1) (d2) (d3) (d4))))
	(declare-datatypes () ((Source (s1) (s2) (s3) (s4))))
	(declare-fun f_allocate (Source) Destination)
	(assert (forall ((x Source)) (exists ((y Destination)) (= (f_allocate x) y))))
	(set-option :opt.priority pareto)
        (declare-const x Int)
        (maximize x)
	(check-sat)
        (get-objectives)
