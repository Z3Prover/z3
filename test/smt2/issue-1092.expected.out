WARNING: optimization with quantified constraints is not supported
(error "line 11 column 11: unbounded objectives on quantified constraints is not supported")
(objectives
 (x oo)
)
